### PR TITLE
Image Caching

### LINKED ISSUE #198: Images aren't being cached
Blog images aren't being cached browserside.

### PR DESCRIPTION
**Pull request by Jordan141**
fixes #198

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -96,7 +97,13 @@ app.use(fileUpload({
     abortOnLimit: true
 }))
 
-app.use(express.static(__dirname + '/public/'))
+app.use(express.static(path.join(__dirname, 'public', 'assets'), {maxAge: '30d'}))
+app.use(express.static(path.join(__dirname, 'public')))
+
+app.use((req, res, next) => {
+    res.set('Cache-Control', 'public', 'max-age=0, must-revalidate')
```
Comment (michaelKurowski) [inline app/app.js:104]: Did you test whether it works?

Comment (michaelKurowski) [reply app/app.js:104]: You can change category image to check

**Review by michaelKurowski — CHANGES_REQUESTED** [review]

**Comment by michaelKurowski** [comment]
I've came up with a solution that wouldn't even need http request if anything.
When you create a link for an image, before adding it to mongo, you just add `?version=checksumOfImage`.
It's just that, the content is cached per url, despite the fact that this param won't be doing anything, it will change url, if the content of the image will change, forcing browser to download new image and cache the new version.
@Jordan141

**Comment by Jordan141** [comment]
Fixes #297

**Comment by michaelKurowski** [comment]
Is the website still working properly? 
You cached all endpoints which means that all responses from server might be ignored by browser

**Pull request merged**